src/common/consts.ts

```diff
@@ -1,50 +1,42 @@
 import { SupportedChainId } from './chains'
-import contractNetworks from '@cowprotocol/contracts/networks.json'
```

Comment:
I assume you are doing this to reduce the lib size?

In one hand, I see it might be an unnecessary bloat. On the other, it's the source of truth.
Ok, right now all the addresses happen to be the same, and we have a limit number of networks

https://github.com/cowprotocol/contracts/blob/main/networks.json#L67-L87

But it's not guaranteed to have always the same address AFAICT, and we might start adding more networks faster in the future.

Still, I see we still depend on the contracts package. Then, what's the benefit of hardcoding the contract address?

Reply:
`@cowprotocol/contracts` is not updated yet unfortunately, the latest version doesn't contain config for Sepolia.
It is the main reason why I did this change.

Reply:
>But it's not guaranteed to have always the same address AFAICT, and we might start adding more networks faster in the future.

I promise I will change it here once we have a case :)

Reply:
Hmmm.
In that case, can you make this change in a way that indicates it's only temporary until the contracts package is not up to date?
A TODO, commented out original code, just so it can be brought back in sync later on.

Reply:
Oh, your reply came while I was writing mine :)
![image](https://github.com/cowprotocol/cow-sdk/assets/43217/11f9ae4f-109b-490d-bf53-025b8e315448)

Sure, I've created an issue to track it https://github.com/cowprotocol/cow-sdk/issues/190